.storybook/recipes/PrimaryNav/PrimaryNav.stories.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import { PrimaryNav } from './PrimaryNav';
 
 export default {
-  title: 'Components/PrimaryNav',
+  title: 'Recipes/PrimaryNav',
```

Comment:
@booc0mtaco snuck this one in here since it's in the `recipes` folder... there's prob low value in having this as a recipe but didn't want to remove yet

Reply:
oh good catch. while you're in there, see if any others in the recipe folder are showing up where they shouldn't!

Reply:
Not that I saw, but I did leave some comments in the component audit spreadsheet 👍